src/test/unit/views/hearing-list.ts

```diff
@@ -0,0 +1,75 @@
+import { expect } from 'chai';
+import request from 'supertest';
+import moment from 'moment';
+
+import { app } from '../../../main/app';
+
+const PAGE_URL = '/hearing-list?courtId=2';
+
+let htmlRes: Document;
+
+describe('List page', () => {
+  beforeAll(async () => {
+    await request(app).get(PAGE_URL).then(res => {
+      htmlRes = new DOMParser().parseFromString(res.text, 'text/html');
+    });
+  });
+
+  it('Check that back button exists', () => {
+    const header = htmlRes.getElementsByClassName('govuk-back-link');
```

Comment:
all your const's are `header` but this one is back button

Reply:
Good spot :) I've updated these now